app/services/transactions/ratings.js

```diff
@@ -0,0 +1,44 @@
+const { sequelize } = require('../../models');
+const WeetsService = require('../../services/weets');
+const RatingsService = require('../../services/ratings');
+const UsersService = require('../../services/users');
+const { RESOURCE_DOES_NOT_EXIST } = require('../../../config/constants');
+const errors = require('../../errors');
+
+const rateWeet = async ({ weetId, userId, score }) => {
```

Comment:
This function is too large, you need to divide it into different functions. Then the complexity will be reduced...
Check [code metrics](
https://marketplace.visualstudio.com/items?itemName=kisstkondoros.vscode-codemetrics) extension

Reply:
Ok, great extension.